aikau/src/main/resources/alfresco/renderers/Thumbnail.js

```diff
@@ -122,6 +150,115 @@ define(["dojo/_base/declare",
       customClasses: "",
       
       /**
+       * The name of the folder image to use. Valid options are: "folder-32.png", "folder-48.png", "folder-64.png"
+       * and "folder-256.png". The default is "folder-64.png".
+       *
+       * @instance
+       * @type {string}
+       * @default
+       */
+      folderImage: "folder-64.png",
+
+      /**
+       * The property to use for the image id. Defaults to "jsNode.nodeRef.nodeRef"
```

Comment:
We don't need to put in the comment what it defaults to, as it is (a) written just below this in the JSDoc because of the @default tag and (b) it can rust